playbooks/gluster-cluster/cluster-vars-4x2.yml

```diff
@@ -0,0 +1,62 @@
+---
+gluster_infra_disktype: RAID6
+gluster_infra_diskcount: 12
+gluster_infra_stripe_unit_size: 128
+
+# The list has one item, if more disks are to be used, append to the list
+gluster_infra_volume_groups:
+  - vgname: vg_vdb
+    pvname: /dev/vdb
+  - vgname: vg_vdc
+    pvname: /dev/vdc
+
+gluster_infra_thinpools:
+  - vgname: 'vg_vdb'
+    thinpoolname: 'thinpool_vg_vdb'
+    thinpoolsize: '45G'
+    poolmetadatasize: '1G'
+  - vgname: 'vg_vdc'
+    thinpoolname: 'thinpool_vg_vdc'
+    thinpoolsize: '45G'
+    poolmetadatasize: '1G'
+
+gluster_infra_lv_logicalvols:
+  - vgname: vg_vdb
+    thinpool: thinpool_vg_vdb
+    lvname: gluster_lv_disk1
+    lvsize: 200G
+  - vgname: vg_vdc
+    thinpool: thinpool_vg_vdc
+    lvname: gluster_lv_disk2
+    lvsize: 200G
+
+# Mount the devices
+gluster_infra_mount_devices:
+  - { path: '/gluster_bricks1', vgname: vg_vdb, lvname: gluster_lv_disk1 }
+  - { path: '/gluster_bricks2', vgname: vg_vdc, lvname: gluster_lv_disk2 }
+
+gluster_infra_fw_ports:
+  - 2049/tcp
+  - 54321/tcp
+  - 5900/tcp
+  - 5900-6923/tcp
+  - 5666/tcp
+  - 16514/tcp
+gluster_infra_fw_permanent: true
+gluster_infra_fw_state: enabled
+gluster_infra_fw_zone: public
+gluster_infra_fw_services:
+  - glusterfs
+
+
+# Variable for creating a GlusterFS volume
+# groups['all'] tells to use the hosts mentioned in the inventory files to be
+# used in the cluster. If only a subset has to be used change the `all' to the
+# name of the section
+
+gluster_cluster_hosts: "{{ groups['all'] }}"
+gluster_cluster_volume: testvol
+gluster_cluster_transport: 'tcp'
+gluster_cluster_force: 'yes'
+gluster_cluster_bricks: '/gluster_bricks1/testvol,/gluster_bricks2/testvol'
```

Comment:
an explanation here about the gluster_cluster_bricks?

Reply:
Ack.